content/logs/explorer/patterns.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Log Patterns (Beta)
+kind: documentation
+description: "Spot Log Patterns"
+aliases:
+  - /logs/patterns
+further_reading:
+- link: "logs/processing"
+  tag: "Documentation"
+  text: Discover how to process your logs
+---
+
+## Overview
+
+Investigating into large volume of logs is really time consuming, you can spend hours in it and still understand a fraction of it. Quite luckily, applicative logs are repetitive and pattern can be detected to automatically bring structure to the problem and help you to quickly see what matters or exclude what's pollute the general understanding.
+
+{{< img src="logs/explorer/patterns/log_patterns_nav.png" alt="Navigate to Log Patterns" responsive="true" style="width:50%;">}}
+
+The Patterns view makes it very helpful to detect error pattern that could be hidden by a noisy recurring error. It shows the number of logs matching a pattern splitted by service and status.
```

Comment:
we use error twice in the first sentence. could we avoid reputation? eg:

"The Patterns view is helpful for detecting and filtering noisy error patterns that might cause you to miss other issues".